core/fx-orm.js

```
@@ -0,0 +1,939 @@
+    // SYNCHRONOUS select - returns immediately
+    select(tableName, query = 'select("*")', connectionId = 'default') {
+        // Find best matching table
+        const connection = this.connector.getConnection(connectionId);
+        const tableMatch = LevenshteinMatcher.findBestMatch(tableName, connection.adapter.tables || [tableName]);
+        
+        if (!tableMatch) {
+            console.warn(`FX[ORM]: No exact table match for "${tableName}", using as-is`);
+            // Use the provided name if no fuzzy match found
+            var matchedTable = tableName;
+        } else {
+            var matchedTable = tableMatch.table;
```

Biome (1.9.4)

[error] Shouldn't redeclare 'matchedTable'. Consider to delete it or rename it.

'matchedTable' is defined here:


(lint/suspicious/noRedeclare)